Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```diff
@@ -0,0 +1,208 @@
+{
+    "author": "Gavin Aguiar",
+    "name": "EventHub trigger",
+    "description": "$EventHubTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with EventHubTrigger function",
+            "type": "CreateNewApp",
+            "inputs" : [
+                {
+                    "assignTo": "$(APP_FILENAME)",
+                    "paramId": "app-fileName",
+                    "defaultValue": "function_app.py",
+                    "required": true
+                },
+                {
+                    "assignTo": "$(FUNCTION_NAME_INPUT)",
+                    "paramId": "trigger-functionName",
+                    "required": true,
+                    "defaultValue": "EventHubTrigger"
+                },
+                {
+                    "assignTo": "$(EVENTHUB_NAME_INPUT)",
+                    "paramId": "eventhubTrigger-eventhubName",
+                    "required": true,
+                    "defaultValue": "myeventhub"
+                },
+                {
+                    "assignTo": "$(CONNECTION_STRING_INPUT)",
+                    "paramId": "eventhubTrigger-connectionString",
+                    "required": true,
+                    "defaultValue": ""
+                }
+            ],
+            "actions": [
+                "readFileContent_FunctionApp",
+                "writeFile_FunctionApp",
+                "showMarkdownPreview"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the main file",
+            "type": "AppendToFile",
+            "inputs" : [
+                {
+                    "assignTo": "$(FUNCTION_NAME_INPUT)",
+                    "paramId": "trigger-functionName",
+                    "required": true,
+                    "defaultValue": "EventHubTrigger"
+                },
+                {
+                    "assignTo": "$(EVENTHUB_NAME_INPUT)",
+                    "paramId": "eventhubTrigger-eventhubName",
+                    "required": true,
+                    "defaultValue": "myeventhub"
+                },
+                {
+                    "assignTo": "$(CONNECTION_STRING_INPUT)",
+                    "paramId": "eventhubTrigger-connectionString",
+                    "required": true,
+                    "defaultValue": ""
+                }
+            ],
+            "actions": [
+                "readFileContent_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "type": "CreateNewBlueprint",
+            "inputs" : [
+                {
+                    "assignTo": "$(BLUEPRINT_FILENAME)",

```

Comment:
We are assuming that value of BLUEPRINT_FILENAME would be without the file extension